install.sh

```diff
@@ -124,20 +116,21 @@ check_root() {
 }
 
 fetch_html() {
-    if ! command -v wget >/dev/null 2<&1; then
-        echo "Error: wget not available"
+    apk add curl >/dev/null 2<&1 || apt-get install curl -y >/dev/null 2<&1
+    if ! command -v curl >/dev/null 2<&1; then
+        echo "Error: curl not available"
         return 1
     fi
 
-    if ! [ -d "public_html" ]; then
-        echo "Error: public_html directory not found"
-        return 1
+    if ! [ -f "public_html/index.html" ]; then
+        curl https://raw.githubusercontent.com/lncm/invoicer-ui/master/dist/index.html \
+        --create-dirs -o public_html/index.html
+        return 0
     fi
 
     if ! [ -f "public_html/index.html" ]; then
```

Comment:
This never happens.

Reply:
I've removed the `return` statement. While a bare return statement takes the return value of the last command, the `fetch_html` function will produce the same value implicitly